WeakAurasOptions/OptionsFrames/OptionsFrame.lua

```diff
@@ -1062,11 +1062,9 @@ function OptionsPrivate.CreateFrame()
 
   frame.ClearPicks = function(self, noHide)
     OptionsPrivate.Private.PauseAllDynamicGroups()
-    if type(frame.pickedDisplay) == "string" then
-      displayButtons[frame.pickedDisplay]:ClearPick(noHide)
-    else
-      for id, button in pairs(displayButtons) do
-        button:ClearPick(true)
+    for id, button in pairs(displayButtons) do
```

Comment:
it seems to break MoverSizer feature

Reply:
import https://wago.io/luckyoneRogue/26 on non-rogue character
open /wa
display this group auras
close /wa
the group is still shown and spam errors
```
1261x [string "Error in: Rogue - Combo Points':'translate ..."]:3: attempt to perform arithmetic on field 'cloneId' (a string value)
[string "=[C]"]: in function `xpcall'
[string "@WeakAuras\Animations.lua"]:76: in function <WeakAuras\Animations.lua:15>
```

Reply:
I fixed the other ones. The new icon is indeed better.

Reply:
> I can't reproduce either of these problems.

Can't reproduce anymore, possible it was because of an error in companion addon fixed since https://github.com/WeakAuras/WeakAuras-Companion/commit/7d6cb9598edd4bfe300cb81b0ad5d27dedf5b994